website/project/model.py

```diff
@@ -1641,6 +1676,7 @@ def use_as_template(self, auth, changes=None, top_level=True):
         )
 
         new.save(suppress_log=True)
+        new.add_contributor(contributor=auth.user, permissions=CREATOR_PERMISSIONS, log=False, save=False)
```

Comment:
When a "copy project structure" (create a new project from a template), it seems I already appear as a listed contributor, even without this line.  (eg Line 1657)

Can you clarify why this change is necessary?